dist/pom.xml

```diff
@@ -409,6 +409,42 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Dist.jar will be created during every buildver=spark-shim, should we unpack spark-rapids-jni/jucx only when building base-spark shim?

Reply:
For build `mvn package -Dbuildver=330` which is base-spark shim, also needs the unpack.